lib/ruby_lsp/ruby_lsp_rails/definition.rb

```diff
@@ -145,7 +145,7 @@ def collect_definitions(name)
 
         methods.each do |target_method|
           location = target_method.location
-          file_path = target_method.file_path
+          file_path = T.must(target_method.file_path)
```

Comment:
Let's improve this entire `each` block. Since we now store the URIs in entries, the code can be simplified. There's no need to instantiate a new URI. Additionally, using the URI directly means this code will work when we start indexing unsaved files with no modifications.

Also, we forgot to use our own helpers to create the range.

```ruby
methods.each do |target_method|
  @response_builder << Interface::Location.new(
    uri: target_method.uri,
    range: range_from_location(target_method.location)
  )
end
```

Reply:
Updated